org.civicrm.reqangular/src/common/modules/settings.js

```diff
@@ -0,0 +1,7 @@
+define([
+    'common/angular'
+], function (angular) {
+    'use strict';
+
+    return angular.module('common.settings', []);
```

Comment:
(Same thing for the bundle) Why a specific module? Do you see the possibility of a collection of various components all related to a `settings` theme?